src/applications/disability-calculator/components/RatingRow.jsx

```diff
@@ -50,18 +49,17 @@ const RatingRow = ({
         />
       </div>
       <div className="vads-l-col--3">
-        {canDelete && (
-          <div>
-            <button
-              type="button"
-              onClick={handleRemoveRating(indx)}
-              className="va-button-link delete-btn vads-u-margin--1p5"
-            >
-              <i className="fas fa-trash-alt vads-u-padding-right--0p5" />
-              Delete
-            </button>
-          </div>
-        )}
+        <div>
+          <button
+            type="button"
+            onClick={handleRemoveRating(indx)}
+            className="va-button-link delete-btn vads-u-margin--1p5"
```

Comment:
The button disabling is working, but there is no visual indication that the buttons are disabled.  Add a CSS style to the disabled state to gray it out.  Something like this should work:

```css
.delete-btn[disabled] {
    opacity: 0.5;
}
```